Content.Shared/_CM14/Xenos/Pheromones/XenoFrenzyPheromonesComponent.cs

```diff
@@ -14,4 +16,16 @@ public sealed partial class XenoFrenzyPheromonesComponent : Component
 
     [DataField]
     public FixedPoint2 Multiplier;
+
+    [DataField]
+    public float AttackDamageModifier = 1.1f;
+
+    [DataField]
+    public FixedPoint2 MovementSpeedModifier = 0.1;
+
+    [DataField(customTypeSerializer: typeof(PrototypeIdListSerializer<DamageGroupPrototype>))]
+    public List<string> DamageTypes = new () { "Blunt", "Slash", "Piercing" };
```

Comment:
This should be DamageTypePrototype, not DamageGroupPrototype (Brute is a group, Blunt/Slash/Piercing are types) 
ProtoId<DamageTypePrototype> can be used instead so you don't need to use a custom type serializer.
You might need to update some usages after this is done.

```suggestion
    [DataField]
    public List<ProtoId<DamageTypePrototype>> DamageTypes = new () { "Blunt", "Slash", "Piercing" };
```